packages/account/src/Sections/Security/Passwords/deriv-email.tsx

```diff
@@ -92,11 +90,4 @@ const DerivEmail = observer(({ email, social_identity_provider, is_social_signup
     );
 });
 
-DerivEmail.propTypes = {
-    email: PropTypes.string,
-    is_dark_mode_on: PropTypes.bool,
-    is_social_signup: PropTypes.bool,
-    social_identity_provider: PropTypes.string,
-};
-
-export default withRouter(DerivEmail);
```

Comment:
is it expected? we don’t need here withrouter?

Reply:
Yes. It is not needed as we are not using any props passed by withRouter function